libs/shared/settings.ts

```diff
@@ -9,13 +9,15 @@ export interface Settings {
   last_visit?: string
   locale: Locale
   injection?: string
+  editor_width: string;
 }
 
 export const DEFAULT_SETTINGS: Settings = Object.freeze({
   split_sizes: [30, 70],
   daily_root_id: ROOT_ID,
   sidebar_is_fold: false,
   locale: Locale.EN,
+  editor_width: 'max-w-prose',
```

Comment:
Not sure how future-proof you want this. Alternatively I could encapsulate this in a enum so the specific classnames can be edited later?